src/main/java/at/hannibal2/skyhanni/features/chat/RareDropMessages.kt

```diff
@@ -4,16 +4,48 @@ import at.hannibal2.skyhanni.SkyHanniMod
 import at.hannibal2.skyhanni.events.LorenzChatEvent
 import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.LorenzUtils.colorCodeToRarity
-import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatchers
 import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
 import net.minecraft.util.ChatComponentText
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 class RareDropMessages {
```

Comment:
Include the word Pet in the class name i guess?

Reply:
i was planning on expanding this class to also have some other features i had in mind related to rare drop messages, thats why i named it like that. if its needed i can just rename it to PetDropMessages for now and change it in the future when i get around to doing that

Reply:
No big thing either way. If you plan on adding non-pet stuff later, then there is no need to rename it twice.